cmd/main.go

```diff
@@ -23,17 +24,17 @@ func doSelect(mb gosql.Backend, slct *gosql.SelectStatement) error {
 	}
 
 	table := tablewriter.NewWriter(os.Stdout)
-	header := []string{}
-	for _, col := range results.Columns {
-		header = append(header, col.Name)
+	header := make([]string, len(results.Columns))
```

Comment:
I'm not a big fan of this style where it's not necessary because it's slightly more bug-prone than just appending.

If you revert these changes I'll approve :+1:

Reply:
Alright. What about using `make([]string, 0, len(results.Columns))` and `append()`?

Reply:
I'd say it's especially not necessarily in the REPL since this aspect of it will never be a performance bottleneck.

You may have good suggestions here, I'd just like to keep the code as simple as possible until there's a good reason to optimize.

Reply:
Alright, I'll revert it.